slices/apparmor.yaml

```diff
@@ -0,0 +1,199 @@
+      /etc/apparmor.d/ch-run:
+      /etc/apparmor.d/chrome:
+      /etc/apparmor.d/code:
+      /etc/apparmor.d/crun:
+      /etc/apparmor.d/devhelp:
+      /etc/apparmor.d/element-desktop:
+      /etc/apparmor.d/epiphany:
+      /etc/apparmor.d/evolution:
+      /etc/apparmor.d/firefox:
+      /etc/apparmor.d/flatpak:
+      /etc/apparmor.d/geary:
+      /etc/apparmor.d/github-desktop:
+      /etc/apparmor.d/goldendict:
+      /etc/apparmor.d/ipa_verify:
+      /etc/apparmor.d/kchmviewer:
+      /etc/apparmor.d/keybase:
+      /etc/apparmor.d/lc-compliance:
+      /etc/apparmor.d/libcamerify:
+      /etc/apparmor.d/linux-sandbox:
+      /etc/apparmor.d/local/README:
+      /etc/apparmor.d/loupe:
+      /etc/apparmor.d/lsb_release:
+      /etc/apparmor.d/lxc-attach:
+      /etc/apparmor.d/lxc-create:
+      /etc/apparmor.d/lxc-destroy:
+      /etc/apparmor.d/lxc-execute:
+      /etc/apparmor.d/lxc-stop:
+      /etc/apparmor.d/lxc-unshare:
+      /etc/apparmor.d/lxc-usernsexec:
+      /etc/apparmor.d/mmdebstrap:
+      /etc/apparmor.d/msedge:
+      /etc/apparmor.d/nautilus:
+      /etc/apparmor.d/notepadqq:
+      /etc/apparmor.d/nvidia_modprobe:
+      /etc/apparmor.d/obsidian:
+      /etc/apparmor.d/opam:
+      /etc/apparmor.d/opera:
+      /etc/apparmor.d/pageedit:
+      /etc/apparmor.d/plasmashell:
+      /etc/apparmor.d/podman:
+      /etc/apparmor.d/polypane:
+      /etc/apparmor.d/privacybrowser:
+      /etc/apparmor.d/qcam:
+      /etc/apparmor.d/qmapshack:
+      /etc/apparmor.d/qutebrowser:
+      /etc/apparmor.d/rootlesskit:
+      /etc/apparmor.d/rpm:
+      /etc/apparmor.d/rssguard:
+      /etc/apparmor.d/runc:
+      /etc/apparmor.d/sbuild:
+      /etc/apparmor.d/sbuild-abort:
+      /etc/apparmor.d/sbuild-adduser:
+      /etc/apparmor.d/sbuild-apt:
+      /etc/apparmor.d/sbuild-checkpackages:
+      /etc/apparmor.d/sbuild-clean:
+      /etc/apparmor.d/sbuild-createchroot:
+      /etc/apparmor.d/sbuild-destroychroot:
+      /etc/apparmor.d/sbuild-distupgrade:
+      /etc/apparmor.d/sbuild-hold:
+      /etc/apparmor.d/sbuild-shell:
+      /etc/apparmor.d/sbuild-unhold:
+      /etc/apparmor.d/sbuild-update:
+      /etc/apparmor.d/sbuild-upgrade:
+      /etc/apparmor.d/scide:
+      /etc/apparmor.d/signal-desktop:
+      /etc/apparmor.d/slack:
+      /etc/apparmor.d/slirp4netns:
+      /etc/apparmor.d/steam:
+      /etc/apparmor.d/stress-ng:
+      /etc/apparmor.d/surfshark:
+      /etc/apparmor.d/systemd-coredump:
+      /etc/apparmor.d/thunderbird:
+      /etc/apparmor.d/toybox:
+      /etc/apparmor.d/trinity:
+      /etc/apparmor.d/tunables/alias:
+      /etc/apparmor.d/tunables/apparmorfs:
+      /etc/apparmor.d/tunables/dovecot:
+      /etc/apparmor.d/tunables/etc:
+      /etc/apparmor.d/tunables/global:
+      /etc/apparmor.d/tunables/home:
+      /etc/apparmor.d/tunables/home.d/site.local:
+      /etc/apparmor.d/tunables/home.d/ubuntu: {text: FIXME, mutable: true}
+      /etc/apparmor.d/tunables/home.d/ubuntu.tmp:
```

Comment:
same comment as below for `/etc/apparmor.d/tunables/xdg-user-dirs.d/site.local.tmp:`